test/fbe/github/graphql/test_client.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# MIT License
+#
+# Copyright (c) 2024 Zerocracy
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+require 'minitest/autorun'
+require 'judges/options'
+require 'webmock/minitest'
+require 'loog'
+require_relative '../../../../lib/fbe/github'
+
+# Test.
+# Author:: Yegor Bugayenko (yegor256@gmail.com)
+# Copyright:: Copyright (c) 2024 Zerocracy
+# License:: MIT
+class TestGitHubGraphQL < Minitest::Test
+  def test_simple_query
+    skip # it's a "live" test, run it manually if you need it
+    WebMock.allow_net_connect!
+    client = Fbe::GitHub::GraphQL::Client.new(token: ENV.fetch('GITHUB_TOKEN', nil))
+    result = client.query(
+      <<-GRAPHQL
+        query {
+          viewer {
+              login
+          }
+        }
+      GRAPHQL
+    )
+    assert(!result.viewer.login.empty?)
```

Comment:
It is [better to use](https://minitest.rubystyle.guide/#refute-false) `refute(false)`  instead `assert(!true)`.